code/_onclick/hud/alert.dm

```diff
@@ -51,13 +51,17 @@
 	thealert.owner = src
 
 	if(new_master)
-		var/old_layer = new_master.layer
-		var/old_plane = new_master.plane
-		new_master.layer = FLOAT_LAYER
-		new_master.plane = FLOAT_PLANE
-		thealert.add_overlay(new_master)
-		new_master.layer = old_layer
-		new_master.plane = old_plane
+		// This should be made into a generic helper proc later tbh
+		var/mutable_appearance/master_appearance = new
```

Comment:
my perfectionism made me unsure on how to make a generic helper proc out of this - I'm unsure what to name it and what arguments there should be / what to name said arguments (besides the original holder of the appearance to copy, ofc) - like, should pixel_x/y/z resetting be toggleable? idk. perfectionism moment.

Reply:
my gut is most uses of this want to do bespoke shit so helpers would not be all that helpful, but I'd go off what actually gets duplicated in future. no sense planning for what has not yet come.